### PR TITLE
hack to solve php 7.4

### PR DESCRIPTION
**Pull request by christiansipola**
https://kaustik.atlassian.net/browse/IN-498
and still support php 7.2

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -53,8 +53,10 @@
         }
         'Ubuntu': {
           case $::operatingsystemrelease {
-            /^(16)/: {
-              $newrelic_php_conf_dir  = ['/etc/php/7.2/mods-available']
+            /^(16|18|20)/: {
```
Comment (christiansipola) [inline manifests/params.pp:56]: it looks like the path structure is the same in all ubuntu

File: manifests/params.pp [hunk 2]
```diff
@@ -53,8 +53,10 @@
...
+              $config_dir = hiera('php::config_dir')
+              notice($config_dir)
```
Comment (christiansipola) [inline manifests/params.pp:58]:
```suggestion
```

**Comment by christiansipola** [comment]
seems to work on a local cloudweb99 with php newrelic activated.

**Comment by christiansipola** [comment]
works for both php 7.2 and php 7.4 on a localcloudweb99

**Review by djerveren — APPROVED** [review]

**Pull request merged**